manifests/init.pp

```diff
@@ -130,6 +130,12 @@
 #   available, likely results in performance degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should
 #   be done incrementally with benchmarking at each step to determine an optimal value for your deployment.
 #
+# @param service_enable
+#   Enable/disable Pulp services at boot.
+
```

Comment:
This stray blank line would cause puppet-strings to only 'see' the lines below.

Reply:
Doh - thanks